app/dashboard/templates/shared/profile_activities_bounty.html

```diff
@@ -1,4 +1,4 @@
-<div class="row result bounty_row {{ bounty.network }}" {% if bounty.github_url %}href="{{ bounty.github_url }}"{% endif %}>
+<a class="row result bounty_row {{ bounty.network }}" target="_blank" href="{{ bounty.url }}">
```

Comment:
@mbeacom Right now when we click on the bounty link -> it redirects to the github page. Shouldn't it redirect to the bounty_details page ?